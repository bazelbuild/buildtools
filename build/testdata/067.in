manifest = [("a",  "b",  "c"), ("ab", "bc", "cd"),]