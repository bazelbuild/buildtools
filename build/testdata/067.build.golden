load(
    # A
    "foo.bzl",
    # B
    "bar",
)
load(":a", "b")

cc_binary(
    # A
    name = "bin",
    # B
    srcs = ["bin.cc"],
    # C
)

cc_binary(
    name = "wibble",
    srcs = ["wibble.cc"],
)

my_list = [
    1,
    # A
    2,
    # B
]

my_1tuple = (
    # A
    1,
    # B
)

my_2tuple = (
    # A
    1,
    # B
    2,
    # C
)

my_dict = {
    "a": 1,
    # A
    "b": 2,
    # B
}

func(a)

func(b)

func(c, d)

func(e, f)
